tmt/steps/provision/testcloud.py

```diff
@@ -342,6 +345,11 @@ class TestcloudGuestData(tmt.steps.provision.GuestSshData):
         option=('-a', '--arch'),
         choices=['x86_64', 'aarch64', 's390x', 'ppc64le'],
         help="What architecture to virtualize, host arch by default.")
+    virtiofs: str = field(
+        default="",
```

Comment:
Would it mae sense to make it `multiple=True` from the beginning? It would require `default_factory=list` and possibly custom validator, and maybe also a dataclass to hold paths. You mentioned multiple mounts would be doable, would this help you implement it? And are there any blockers? So far it looks very cheap to add, but I don’t know virtiofs, so no push.

How about different name? As a user, I’m not really interested in virtiofs as a technology, but, I would be very interested in *mounting* directories - and there are also other plugins, I bet `container` could support *mounts* too, so how about something more general, like, hm, `mount`? :)

Any  hance of needing some kind of flags? Docker and podman have some, `z` and `Z` at least, does virtiofs have something similar?

Reply:
> Would it mae sense to make it `multiple=True` from the beginning? It would require `default_factory=list` and possibly custom validator, and maybe also a dataclass to hold paths. You mentioned multiple mounts would be doable, would this help you implement it? And are there any blockers? So far it looks very cheap to add, but I don’t know virtiofs, so no push.
> 
> How about different name? As a user, I’m not really interested in virtiofs as a technology, but, I would be very interested in _mounting_ directories - and there are also other plugins, I bet `container` could support _mounts_ too, so how about something more general, like, hm, `mount`? :)
> 
> Any hance of needing some kind of flags? Docker and podman have some, `z` and `Z` at least, does virtiofs have something similar?



> Would it mae sense to make it `multiple=True` from the beginning? It would require `default_factory=list` and possibly custom validator, and maybe also a dataclass to hold paths. You mentioned multiple mounts would be doable, would this help you implement it? And are there any blockers? So far it looks very cheap to add, but I don’t know virtiofs, so no push.
> 
> How about different name? As a user, I’m not really interested in virtiofs as a technology, but, I would be very interested in _mounting_ directories - and there are also other plugins, I bet `container` could support _mounts_ too, so how about something more general, like, hm, `mount`? :)
How about the name `shared-fs`?
> 
> Any hance of needing some kind of flags? Docker and podman have some, `z` and `Z` at least, does virtiofs have something similar?